### PR TITLE
Fix spacing issues in SBRP

### PR DESCRIPTION
**Pull request by dseefeld**
The generator has been updated to do better at spacing in the projects
it generates.  When generating new source, if these are re-generated,
the always show up as differences.  Making this change to reduce
unneccessary differences.

### CODE REVIEW

**Review by MichaelSimons — APPROVED** [review]

File: src/referencePackages/src/system.runtime.interopservices/4.3.0/System.Runtime.InteropServices.4.3.0.csproj [hunk 1]
```diff
@@ -15,48 +16,61 @@
...
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.5' ">
+    <PackageReference Include="Microsoft.NETCore.Platforms" Version="1.1.0" />
+    <PackageReference Include="Microsoft.NETCore.Targets" Version="1.1.0" />
+    <PackageReference Include="System.Reflection" Version="4.3.0" />
+    <PackageReference Include="System.Reflection.Primitives" Version="4.3.0" />
+    <PackageReference Include="System.Runtime" Version="4.3.0" />
+    <PackageReference Include="System.Runtime.Handles" Version="4.3.0" />
+  </ItemGroup>
+
```
Comment (crummel) [inline src/referencePackages/src/system.runtime.interopservices/4.3.0/System.Runtime.InteropServices.4.3.0.csproj:52]: Non-whitespace change - just want to make sure this is intentional.

File: src/referencePackages/src/system.security.permissions/4.7.0/System.Security.Permissions.4.7.0.csproj [hunk 2]
```diff
@@ -16,24 +17,29 @@
...
-        <Reference Include="System.ServiceProcess" />
-        <Reference Include="System.Transactions" />
-        <Reference Include="WindowsBase" />
-    </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+    <PackageReference Include="NETStandard.Library" Version="$(NETStandardImplicitPackageVersion)" />
+    <PackageReference Include="System.Security.AccessControl" Version="4.7.0" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp3.0' ">
```
Comment (crummel) [inline src/referencePackages/src/system.security.permissions/4.7.0/System.Security.Permissions.4.7.0.csproj:25]: Non-whitespace change again.

**Review by crummel — APPROVED** [review]
LGTM, just flagged a couple non-whitespace changes that look fine but I didn't know if were intentional.

File: src/referencePackages/src/system.runtime.interopservices/4.3.0/System.Runtime.InteropServices.4.3.0.csproj (hunk 1 above)
Comment (crummel) [reply src/referencePackages/src/system.runtime.interopservices/4.3.0/System.Runtime.InteropServices.4.3.0.csproj:52]: The netcoreapp1.1 section is new, to clarify.

File: src/referencePackages/src/system.security.permissions/4.7.0/System.Security.Permissions.4.7.0.csproj (hunk 2 above)
Comment (crummel) [reply src/referencePackages/src/system.security.permissions/4.7.0/System.Security.Permissions.4.7.0.csproj:25]: Netcoreapp3.0 section is new.

File: src/referencePackages/src/system.runtime.interopservices/4.3.0/System.Runtime.InteropServices.4.3.0.csproj (hunk 1 above)
Comment (dseefeld) [reply src/referencePackages/src/system.runtime.interopservices/4.3.0/System.Runtime.InteropServices.4.3.0.csproj:52]: Thanks for catching the added section.  It was not intentional.

File: src/referencePackages/src/system.security.permissions/4.7.0/System.Security.Permissions.4.7.0.csproj (hunk 2 above)
Comment (dseefeld) [reply src/referencePackages/src/system.security.permissions/4.7.0/System.Security.Permissions.4.7.0.csproj]: Same

**Pull request merged**